distage/distage-core-api/src/main/scala/izumi/distage/model/definition/Lifecycle.scala

```diff
@@ -1160,7 +1161,18 @@ private[definition] trait LifecycleCatsInstances0 {
   }.asInstanceOf[Monoid[Lifecycle[F, A]]]
 }
 
-private[definition] trait LifecycleIzumiInstances {
+private[definition] trait LifecycleIzumiInstances extends LifecycleIzumiInstancesPriority0
+
+private[definition] trait LifecycleIzumiInstancesPriority0 extends LifecycleIzumiInstancesPriority1 {
+  @inline implicit final def zio3LifecycleInstance = Predefined(functor3ForLifecycle[zio.ZIO])
```

Comment:
@VladPodilnyk 
Are these specializations necessary? Don't zio, IO and monix-bio instances resolve just with `functor2ForLifecycle`/`functor3ForLifecycle` instances?

Reply:
@neko-kai, it's not, I was confused by Intelij's message about `predefined` and added them. My bad, I'll remove them in a sec.

Reply:
Done.